src/object/cli_ec.c

```diff
@@ -57,27 +57,67 @@ struct ec_params {
 	struct ec_params        *next;	/* Pointer to next entry in list. */
 };
 
+struct ec_fetch_params {
+	daos_iod_t		*iods;	/* Replaces iod array in fetch. */
+	unsigned int		 nr;	/* number of records in iods    */
+	daos_iod_t			 niod;
+	struct ec_fetch_params		*next;/* Next entry in list */
+};
+
+static bool
+ec_is_full_stripe(daos_iod_t *iod, struct daos_oclass_attr *oca,
+		  unsigned int recx_idx)
+{
+	unsigned int	ss = oca->u.ec.e_k * oca->u.ec.e_len;
+	unsigned int start = iod->iod_recxs[recx_idx].rx_idx * iod->iod_size;
+	unsigned int length = iod->iod_recxs[recx_idx].rx_nr * iod->iod_size;
```

Comment:
alignment?

Reply:
daos convention is tab aligned variables.   Also one extra space if there are pointers

int<tab>*foo;
int<tab><space>bar;

Reply:
Fixed.